src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -62,9 +62,17 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
 
   const scheduleAnimationCallback = config.scheduleAnimationCallback;
   const scheduleDeferredCallback = config.scheduleDeferredCallback;
+  const useSyncScheduling = config.useSyncScheduling;
 
-  // The default priority to use for updates.
-  let defaultPriority : PriorityLevel = LowPriority;
+  // The priority level to use when scheduling an update.
+  let priorityContext : (PriorityLevel | null) = null;
```

Comment:
Good call. I think I'll just get rid of `defaultPriorityContext` entirely. I added it when I thought we'd need a way to change the default on demand, but now that it's determined by the host config it's not necessary.